### DISCUSSION TITLE
Integrating a new Aqara T2 Spotlight variation into z2m: Seeking guidance on Pull Request Approach [SSWQD03LM]

### DISCUSSION THREAD

**Discussion by yhbyhb** [Koenkk/zigbee2mqtt#21426] [Devices]
I'm trying to integrating a new device into z2m (Zigbee2MQTT), specifically a variation of the [Aqara T2 spotlight](https://www.zigbee2mqtt.io/devices/SSWQD03LM.html). Interestingly, this new device shares the same Zigbee LED driver (SSWQD03LM, as depicted by the provided picture) as the already supported Aqara T2 spotlight, but its zigbeeModel differs.

The existing "zigbeeModel" in z2m for the supported device is listed as `lumi.light.acn026`, while the new device's zigbeeModel is identified as `lumi.light.acn024`. To facilitate the integration process, I've generated an external definition for the new device according to provided [instructions](https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html)
```
const {light} = require('zigbee-herdsman-converters/lib/modernExtend');

const definition = {
    zigbeeModel: ['lumi.light.acn024'],
    model: 'lumi.light.acn024',
    vendor: 'Aqara',
    description: 'Automatically generated definition',
    extend: [light({"colorTemp":{"range":[166,370]}})],
    meta: {},
};

module.exports = definition;
```

Before proceeding with pull requests to add this new device, I'm seeking clarification on the preferred approach. I've contemplated two options:

1.  Submitting a single pull request to [zigbee-herdsman-converters](https://github.com/Koenkk/zigbee-herdsman-converters) , given that both devices utilize the same Zigbee LED driver. This entails updating the existing definitions to include the new device, as shown in the snippet below:
```
diff --git a/src/devices/lumi.ts b/src/devices/lumi.ts
index e00cb02b..e1fe6748 100644
--- a/src/devices/lumi.ts
+++ b/src/devices/lumi.ts
@@ -2274,7 +2274,7 @@ const definitions: Definition[] = [
         extend: [lumiZigbeeOTA(), lumiLight({colorTemp: true})],
     },
     {
-        zigbeeModel: ['lumi.light.acn026'],
+        zigbeeModel: ['lumi.light.acn026', 'lumi.light.acn024'],
         model: 'SSWQD03LM',
         vendor: 'Aqara',
         description: 'Spotlight T2',
```
2.  Creating a pull request to zigbee2mqtt.io first, considering the difference in product models. In the provided picture, the new device's product model is `LTSZNSD02LM`, while the model listed in z2m's product model is `LTSZNTD01LM`.

Your insights and recommendations on the preferred approach would be greatly appreciated!

![IMG_2689](https://github.com/Koenkk/zigbee2mqtt/assets/6398628/b4250773-a47f-40da-9fbc-d7e6fb87a095)

**Comment by Koenkk** [comment]
Both are `SSWQD03LM` so please add it (option 1)
